deployment/src/test/webui/.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+enableImmutableInstalls: false
```

Comment:
I think this all needed because out of the box its using Yarn 2 but it looks like your tests expect Yarn 1 ?